### PR TITLE
[142] Develop a Contact Form Page using HTML

### PR DESCRIPTION
**Pull request by RafiatAmin**
Screen Shot 2020-12-03 at 10 36 33 AM

### CODE REVIEW

**Review by Ireti203 — CHANGES_REQUESTED** [review]
You have 2 contact.html files. You have one in the HTML folder and another in the html-forms folder. Kindly remove one of these files, you only need one.

**Comment by RafiatAmin** [comment]
I have sent a screenshot of my folder already. There’s nothing I can do again. Just pick the one that works. Over a month now. I just want to continue my course please.

File: submissions/rafiatAminu/HTML/html-forms/contact.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
```
Comment (adaugochi) [inline submissions/rafiatAminu/HTML/html-forms/contact.html:1]: Well done on the work so far

Moving forward, you have to remove the file. It is not supposed to be here. You already have it inside the correct folder. So delete this one

**Review by adaugochi — CHANGES_REQUESTED** [review]
So locate this `.DS_Store` inside this path and remove it as well. It is not supposed to be part of your PR
image

**Pull request closed without merging**